setup.py

```diff
@@ -3,7 +3,7 @@
     find_packages,
 )
 
-__version__ = '0.3.0'
+__version__ = '0.4.0'
```

Comment:
Let's increase the bug-fix number (last digit) rather than the minor version number. The rule of thumb is
* Increase the major version number if you are making breaking changes;
* Increase the minor version number if new feature is added; and
* Increase the bug-fix version number otherwise.